pyinteraph/pyinteraph.py

```diff
@@ -125,7 +125,7 @@ def main():
                         default = 0.0, \
                         help = hcperco_helpstr.format(hcperco_default))
 
-    hcdat_default = "hydrophobic-clusters.dat"
```

Comment:
It would be better to add the .csv extension here in the default file name (as well as for the other default file names in the help), as we were doing with the .dat files, in order to show the user the full file name that he should expect if he does not specify otherwise.

Reply:
The default outputs are already .csv (and .dat for the matrix). The print function adds the correct extension depending on the file

Reply:
I fixed this by changing the help string underneath. Otherwise it results in salt-bridges.csv_all_chains.csv since I add the extension in the print function